modules/oadp-restic-issues.adoc

```diff
@@ -99,13 +99,44 @@ $ oc get all -n <namespace>
 [id="restic-backup-cannot-be-recreated-after-s3-bucket-emptied_{context}"]
 == Restic Backup CR cannot be recreated after bucket is emptied
 
-If you create a Restic `Backup` CR for a namespace, empty the S3 bucket, and then recreate the `Backup` CR for the same namespace, the recreated `Backup` CR fails.
+If you create a Restic `Backup` CR for a namespace, empty the object storage/bucket, and then recreate the `Backup` CR for the same namespace, the recreated `Backup` CR fails.
```

Comment:
@RichardHoch 
Change 'recreate' to 're-create' and 'recreated' - to 're-created'.